vendor/assets/stylesheets/components/_roboto.scss

```diff
@@ -1,49 +1,49 @@
 @font-face {
     font-family: "Roboto";
-    src: local(Roboto Thin), url('#{$roboto-font-path}Roboto-Thin.eot');
-    src: url("#{$roboto-font-path}Roboto-Thin.eot?#iefix") format('embedded-opentype'),
-        url("#{$roboto-font-path}Roboto-Thin.woff2") format("woff2"),
-        url("#{$roboto-font-path}Roboto-Thin.woff") format("woff"),
-        url("#{$roboto-font-path}Roboto-Thin.ttf") format("truetype");
+    src: local(Roboto Thin), font-url('#{$roboto-font-path}Roboto-Thin.eot');
+    src: font-url("#{$roboto-font-path}Roboto-Thin.eot?#iefix") format('embedded-opentype'),
+        font-url("#{$roboto-font-path}Roboto-Thin.woff2") format("woff2"),
+        font-url("#{$roboto-font-path}Roboto-Thin.woff") format("woff"),
+        font-url("#{$roboto-font-path}Roboto-Thin.ttf") format("truetype");
 
     font-weight: 200;
 }
 @font-face {
     font-family: "Roboto";
-    src: local(Roboto Light), url('#{$roboto-font-path}Roboto-Light.eot');
-    src: url("#{$roboto-font-path}Roboto-Light.eot?#iefix") format('embedded-opentype'),
-        url("#{$roboto-font-path}Roboto-Light.woff2") format("woff2"),
-        url("#{$roboto-font-path}Roboto-Light.woff") format("woff"),
-        url("#{$roboto-font-path}Roboto-Light.ttf") format("truetype");
+    src: local(Roboto Light), font-url('#{$roboto-font-path}Roboto-Light.eot');
+    src: font-url("#{$roboto-font-path}Roboto-Light.eot?#iefix") format('embedded-opentype'),
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings